site/src/Util/NotificationHydrator.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace App\Util;
+
+/**
+ * This is the data class for creating a notification
+ * which will be then sent out to the RabbitMQ queue
+ *
+ * @copyright Aquib Baig <aquibbaig97@gmail.com>
+ *
+ * Class NotificationHydrator
+ */
+class NotificationHydrator
+{
+    /**
+     * @var string $subject
+     */
+    protected $subject;
+
+    /**
+     * @var string $link
+     */
+    protected $link;
```

Comment:
What's that for?

Reply:
Actually, Mgilet provides us with a `link` field by default which is nullable. So, we can either add/remove this field, later on, based on our usage

Reply:
So let's do it the other way around: remove the link field in our `Notification(Hydrator)` class, as I don't see any use for it. If mgilet uses it that's fine, we can always set it to null.